ml/encoders.py

```diff
@@ -3,7 +3,7 @@
 
 from typing import List
 
-from . import BaseEncoder, BaseDistance
+from . import BaseEncoder, BaseDistance, TensorWithMask
```

Comment:
For now imported but unused...

```suggestion
from . import BaseEncoder, BaseDistance
```